### PR TITLE
CertUtil Adds and Updates

### PR DESCRIPTION
**Pull request by MHaggis**
## New Analytic Stories

- Ingress Tool Transfer
- Deobfuscate/Decode Files or Information

## New Detections
- CertUtil Download With URLCache and Split Arguments (Passed)
- CertUtil Download With VerifyCtl and Split Arguments (Passed)
- CertUtil with Decode Flag  (Passed)

### CODE REVIEW

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

CertUtil Download With URLCache and Split Arguments :white_check_mark:
- diskUsage: 393216
- runDuration: 2.793s
- scanCount: 2040

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

CertUtil Download With VerifyCtl and Split Arguments :white_check_mark:
- diskUsage: 401408
- runDuration: 2.8850000000000002s
- scanCount: 2040

File: detections/endpoint/certutil_download_with_verifyctl_and_split_arguments.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  kill_chain_phases:
+  - Exploitation
+  mitre_attack_id:
+  - T1105
+  product:
+  - Splunk Enterprise
+  - Splunk Enterprise Security
+  - Splunk Cloud
+  required_fields:
+  - _time
```
Comment (patel-bhavin) [inline detections/endpoint/certutil_download_with_verifyctl_and_split_arguments.yml:46]: @MHaggis : can you add other fields used in the search to this?

File: detections/endpoint/certutil_download_with_urlcache_and_split_arguments.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  kill_chain_phases:
+  - Exploitation
+  mitre_attack_id:
+  - T1105
+  product:
+  - Splunk Enterprise
+  - Splunk Enterprise Security
+  - Splunk Cloud
+  required_fields:
+  - _time
```
Comment (patel-bhavin) [inline detections/endpoint/certutil_download_with_urlcache_and_split_arguments.yml:45]: @MHaggis : add other required fields that are used in the search

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

CertUtil With Decode Argument :white_check_mark:
- diskUsage: 397312
- runDuration: 2.371s
- scanCount: 1507

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**